.github/workflows/ci.yml

```
@@ -18,16 +18,16 @@ jobs:
         go-version: [1.22.5]
 
     steps:
-    - uses: actions/checkout@v2
-    - uses: actions/setup-go@v2
+    - uses: actions/checkout@v4
+    - uses: actions/setup-go@v5
       with:
         go-version: ${{ matrix.go-version }}
-    - uses: goreleaser/goreleaser-action@v2
+    - uses: goreleaser/goreleaser-action@v6
```

goreleaser fails.

---

Yes, that's indeed the case. I tried looking into this, but didn't arrive a quick solution. Do you know how this can be fixed?

---

I would revert this change and merge the other upgrades.
```suggestion
    - uses: goreleaser/goreleaser-action@v2
```
Then I would circle back and upgrade goreleaser/goreleaser-action separately.

https://goreleaser.com/errors/version/?h=.goreleaser.yml#unsupported-configuration-version

https://goreleaser.com/blog/goreleaser-v2

---

Good idea, let try that instead

---

* #29 completes the solution.